fastlane/lanes/release.rb

```diff
@@ -457,14 +506,17 @@
 # @param [Boolean] beta Indicate if we should build a beta or regular release
 #
 def trigger_buildkite_release_build(branch:, beta:)
-  buildkite_trigger_build(
+  build_url = buildkite_trigger_build(
     buildkite_organization: BUILDKITE_ORGANIZATION,
     buildkite_pipeline: BUILDKITE_PIPELINE,
     branch: branch,
     environment: { BETA_RELEASE: beta },
     pipeline_file: 'release-builds.yml',
     message: beta ? 'Beta Builds' : 'Release Builds'
   )
+
+  message = "This build triggered a build on <code>#{branch}</code>:<br>- #{build_url}"
+  buildkite_annotate(style: 'info', context: 'trigger-release-build', message: message) if is_ci
```

Comment:
A different nitpick. Given we don't use `message` unless we are in CI:

```suggestion
  return unless is_ci
  
  message = "This build triggered a build on <code>#{branch}</code>:<br>- #{build_url}"
  buildkite_annotate(style: 'info', context: 'trigger-release-build', message: message)
```